upgrade-1.x-to-2.x.sh

```diff
@@ -888,10 +888,10 @@ sync
 if [ "$NOREBOOT" == "no" ]; then
     # Reboot into new OS
     log "Rebooting into new OS in 5 seconds..."
-    progress 100 "ResinOS: update successful, rebooting..."
+    progress 100 "Update successful, rebooting"
     nohup bash -c " /bin/sleep 5 ; /sbin/reboot " > /dev/null 2>&1 &
 else
     log "Finished update, not rebooting as requested."
     log "NOTE: Supervisor and stopped services kept stopped!"
-    progress 100 "ResinOS: update successful."
+    progress 100 "Update successful."
```

Comment:
I would remove the full stop from this too as you seem to have removed them from all the other messages.

Reply:
Yup, that's good.